### PR TITLE
staging-v4.1.0

### PR DESCRIPTION
**Pull request by jasonintju**
* Revert "downtime for runtime release (#680)"
* add mantsjs v2.0.0 relative updates (#580)
* update mantajs to v3.0.0

## Description

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [ ] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [ ] Updated relevant documentation in the code
- [ ] Test code changes manually, and describe your procedure in a comment on this PR
- [ ] Mark this PR with the correct milestone

## If PR to `main`
- [x] Update the version number in `package.json` and `src/config/common.json`
- [x] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [x] Update `CHANGELOG.md`
- [x] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

**Review by jincan39 — APPROVED** [review]
lgtm

File: src/pages/SendPage/SendButton.tsx [hunk 1]
```diff
@@ -63,6 +65,7 @@ const ValidationSendButton = ({ showModal }) => {
   } = useSend();
   const { signerIsConnected, signerVersion } = usePrivateWallet();
   const { externalAccount } = useExternalAccount();
+  const apiIsDisconnected = apiState === API_STATE.ERROR || apiState === API_STATE.DISCONNECTED;
```
Comment (zhangjing28) [inline src/pages/SendPage/SendButton.tsx:68]: I think the api is disconneced when `apiState !== API_STATE.READY`. What's more, I saw the same logic in two different files, why not remove this logic to the `SubstrateContext.file`?

Comment (jasonintju) [reply src/pages/SendPage/SendButton.tsx:68]: That's all what Kevin did before in #382, which is not the most concerned now. I think we can just leave it, it doesn't matter much.

**Review by osangel — APPROVED** [review]

File: src/pages/SendPage/SendButton.tsx (hunk 1 above)
Comment (jasonintju) [reply src/pages/SendPage/SendButton.tsx:68]: > I think the api is disconneced when `apiState !== API_STATE.READY`. What's more, I saw the same logic in two different files, why not remove this logic to the `SubstrateContext.file`?

API_STATE.INIT !== API_STATE.READY but it is not 'disconnected'

**Comment by osangel** [comment]
lgtm

**Pull request merged**